src/Message/AppToHost.elm

```diff
@@ -20,5 +20,5 @@ decodeFromApp =
         [ Decode.map Publish PubSub.publicationDecoder
         , Decode.map Subscribe PubSub.subscribeDecoder
         , Decode.map Unsubscribe PubSub.unsubscribeDecoder
-        , Decode.map NavRequest Navigation.decoder
+        , Decode.map RouteChange (Path.decoder |> expectLabel "routeChange")
```

Comment:
It felt excessive to make a separate `Messages/RouteChange` file for this, but I could be convinced to do so pretty easily.